Alisha/data/lab1.ipynb

```diff
@@ -0,0 +1,354 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #38.        num_points=widgets.Dropdown(options=[50, 100, 200], value=100, description='Points')
</pre><p>looks like changing the points doesnot actually change the number of points in x axis. If that is intended, please add reason below this cell. Else fix the issue</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/suparthghimire/data-analysis-and-visualization-bca-2025/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>